### PR TITLE
xcute: Add documentation

### PR DESCRIPTION
**Pull request by AymericDu**
##### SUMMARY

Add documentation

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

- xcute

##### SDS VERSION

```
openio 5.3.1.dev1
```

### CODE REVIEW

**Comment by murlock** [comment]
IMO we should avoid to add picture / SVG in this repository. 

We should push documentation in dedicated repository

File: oio/xcute/README.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+![Architecture - status](architecture_status.svg)
+
+## Configuration
+
+### Server/orchestrator
+
+Sample configuration for the server/orchestrator:
+
+```yaml
```
Comment (loamhoof) [inline oio/xcute/README.md]: `yaml` ? :smile:

File: oio/xcute/doc/README.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
+# xcute
+
+xcute is a tool that dispatches tasks
```
Comment (loamhoof) [inline oio/xcute/doc/README.md]: Dispatching task is the method, not the goal. I'd rather say that xcute is a tool for executing jobs in a distributed manner.

File: oio/xcute/doc/README.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+in order to process them across the entire platform.
+
+## Terminology
+
+- Job: task series
+- Task: unit of execution
+- Orchestrator: task dispatcher
+- Worker: Task executor
```
Comment (loamhoof) [inline oio/xcute/doc/README.md]: Inconsistent capitalisation: `: Task`

File: oio/xcute/doc/README.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+## Architecture
+
+### Services
+
+For each job processed, the orchestrator will generate all the tasks
+and send them to the workers (even-agent).
```
Comment (loamhoof) [inline oio/xcute/doc/README.md]: `even-agent`

File: oio/xcute/doc/README.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
++----------------------------------+----------+-------------------+---------------+---------------+
+| 20200319164556708006-145b1fbcd5b | RUNNING  | rawx-decommission | 1584636356.71 | 1584636424.03 |
+| 20200319164420713268-60d3d630072 | FINISHED | rawx-decommission | 1584636260.71 | 1584636275.15 |
+| 20200319164221665517-15c22dd56ac | FINISHED | rawx-decommission | 1584636141.67 | 1584636155.09 |
++----------------------------------+----------+-------------------+---------------+---------------+
+```
+
+### Pause a job
+
+Until all tasks are sent, you can pause the job.
```
Comment (loamhoof) [inline oio/xcute/doc/README.md]: Prefer `As long as` to `until`:  
`As long as all its tasks are not sent`

**Pull request merged**